app/models/secret.rb

```diff
@@ -20,7 +20,37 @@ def latest_public_keys account, kind, id
           all.
           map(&:value)
     end
+
+
+   # WITH expired_secrets AS (SELECT resource_id FROM secrets GROUP BY resource_id HAVING max(expires_at) < NOW()) SELECT resource_id, value AS ttl FROM annotations NATURAL JOIN expired_secrets WHERE name = 'ttl'
+
+# Album.group_and_count(:artist_id).having{count.function.* >= 10}
+# # SELECT artist_id, count(*) AS count FROM albums
+# # GROUP BY artist_id HAVING (count(*) >= 10)
+
+    def freshly_expired
```

Comment:
Nitpick: this should be called `pending_expirations`, otherwise it looks like it's a filter on secrets, which it isn't.

Reply:
naming "nitpicks" are always welcomed.  they're important.